### PR TITLE
Update french language

### PR DESCRIPTION
**Pull request by Smeulf**
## Types of changes

- [X] Translation
- [X] I have read the [**CONTRIBUTING**](Contributing.md) document.

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes to Core Features:

* [X] Have you added an explanation of what your changes do and why you'd like us to include them?
* [X] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: resources/language/resource.language.fr_fr/strings.po [hunk 1]
```diff
@@ -315,3 +315,399 @@ msgstr "Vérifier toutes les X minutes si mise à jour programmée"
...
+msgid "Netflix returned an unknown error."
+msgstr "Netflix à renvoyé une erreur inconnue"
+
+msgctxt "#30102"
+msgid "Netflix returned the following error:"
+msgstr "Netflix a renvoyé l'erreur suivante : "
+
+msgctxt "#30103"
+msgid "Error details have been written to the Kodi logfile. If the error message does not provide guidance on how to resolve the issue or you continue to receive this error, please report it via GitHub. Please provide a full debug log with your error report (enable \"Debug Logging\" in Kodi settings)."
+msgstr "Les détails de l'erreur ont été écrits dans le fichier journal de Kodi. Si le message d'erreur ne vous indique pas comment résoudre le problème ou si vous continuez à recevoir cette erreur, signalez-le via GitHub. Veuillez fournir un journal de débogage complet avec votre rapport d'erreur (activez \ "Journalisation du débogage \" dans les paramètres Kodi)."
```
Comment (CastagnaIT) [inline resources/language/resource.language.fr_fr/strings.po]: do not put space between escape char \ and "

File: resources/language/resource.language.fr_fr/strings.po [hunk 2]
```diff
@@ -315,3 +315,399 @@ msgstr "Vérifier toutes les X minutes si mise à jour programmée"
...
+msgid "Discover the latest content added."
+msgstr "Découvrir les derniers contenus ajoutés"
+
+msgctxt "#30147"
+msgid "Next page >>"
+msgstr "Page suivante >>"
+
+msgctxt "#30148"
+msgid "<< Previous page"
+msgstr "Page précédente"
```
Comment (CastagnaIT) [inline resources/language/resource.language.fr_fr/strings.po]: missing "<<" like english version

File: resources/language/resource.language.fr_fr/strings.po [hunk 3]
```diff
@@ -315,3 +315,399 @@ msgstr "Vérifier toutes les X minutes si mise à jour programmée"
...
+msgid "Netflix addon configuration"
+msgstr "Configuration de l'extension Netflix"
+
+msgctxt "#30155"
+msgid "Do you have a Netflix Premium account?"
+msgstr "Avez-vous un compte Netflix Premium ?"
+
+msgctxt "#30156"
+msgid "Does your device support the 4K standard?\r\nThe device must:\r\nBe Netflix certified, support for Widevine Security level L1 and have a 4K display HDCP 2.2"
+msgstr "Votre appareil prend-il en charge la norme 4K ?\rL'appareil doit :\rEtre certifié Netflix, prendre en charge le niveau de sécurité Widevine L1 et disposer d’un écran 4K HDCP 2.2."
```
Comment (CastagnaIT) [inline resources/language/resource.language.fr_fr/strings.po]: \r\n go together not only \r

**Comment by Smeulf** [comment]
Changes mades and pushed :) thks for the review!

**Pull request merged**